package.json

```diff
@@ -53,24 +53,25 @@
     "@babel/core": "^7.5.5",
     "@babel/runtime": "^7.5.5",
     "@react-native-community/eslint-config": "^1.0.0",
+    "@react-native-community/datetimepicker": "./",
```

Comment:
this looks funny, the dev dependency of `@react-native-community/datetimepicker` is `@react-native-community/datetimepicker` - why?

Reply:
because the second paramaters is deprecated applyNativeModulesSettingsGradle(settings, "../..") I add this on package.json to autolink works equall on Android/iOS

Reply:
> because the second paramaters is deprecated applyNativeModulesSettingsGradle(settings, "../..")

ah, interesting, is there any resource (some docs) about this that you can point me to? Thank you!

Reply:
@vonovak there is https://github.com/react-native-community/cli/blob/master/packages/platform-android/native_modules.gradle#L280-L281

Reply:
Sorry I mention the react-native 62 but it is related to `cli`

Reply:
This is shouldn't be necessary. What you're expected to do, is to have `example/package.json` with a `"@react-native-community/datetimepicker": "../"` in its `dependencies`, like a normal app would do. Hope that fixes your issues. Also, remove the `--root` from `run-android`

Reply:
Also, you likely don't need the custom config at all. All you need to do is to run the example from the `example/` directory, like you would do for a normal app.

Check how slider does it: https://github.com/react-native-community/react-native-slider

Reply:
@thymikee thanks for the comment. Looking at the slider repo I see they use a very different setup with yarn workspaces it seems; there is package.json in the root, in src folder and in example folder - seems we'd need to do a bunch of refactors to use that setup and time to understand it.

If I look at a bob-generated library and example, bob also does it differently. There is package.json in the root and in the example folder - but this one does not use "some-dependency": "../" in its `dependencies` - instead `metro.config.js` in the example folder instructs metro to look for JS files in the parent folder (watchFolders: [root],).

I don't mean to complain, but this thing is really much harder than it should be.

I'll just link the native modules manually I guess.

Reply:
@thymikee @vonovak thanks for detailed explanation I'll take a look at the slider repository

Reply:
The slider has a different setup with Yarn workspaces, which would require lots of file shifting in this case. We identified the root cause here: https://github.com/react-native-community/cli/issues/1137#issuecomment-619988897